src/data/api/fetch.ts

```diff
@@ -62,6 +62,27 @@ export const sendEventTilVeilarbperson = async (event: FrontendEvent) => {
     return handterRespons(respons);
 };
 
+export const sendOverblikkFilter = async (event: string[]): Promise<string[]> => {
+    const url = `/veilarbfilter/api/overblikkvisning`;
+    const respons = await fetch(url, createPOSToptions(event));
+
+    return handterRespons(respons);
+};
+
+export const useOverblikkFilter = () => {
+    const { data, error, isLoading, mutate } = useSWR<string[], ErrorMessage>(
+        `/veilarbfilter/api/overblikkvisning`,
+        fetcher,
+        {
+            shouldRetryOnError: false,
+            revalidateIfStale: false,
+            revalidateOnFocus: false,
+            revalidateOnReconnect: false
+        }
```

Comment:
I think these options are configured "globally" for SWR, i.e. we don't have to repeat them here.